app/views/admin/publishers/show.html.slim

```diff
@@ -53,9 +53,9 @@ hr
         .db-info-row
           .db-field = "Overall BAT balance:"
           .db-value = "#{publisher_overall_bat_balance(@publisher)}"
-        - if @publisher.default_currency.present?
+        - if @publisher.uphold_connection.default_currency.present?
           .db-info-row
-            .db-field = "Overall #{@publisher.default_currency} balance:"
+            .db-field = "Overall #{@publisher.uphold_connection. default_currency} balance:"
```

Comment:
extra space in `uphold_connection. `

Reply:
I ended up making the Uphold Connection it's own section on the show page of the administration dashboard, that way it's easier to view 

![image](https://user-images.githubusercontent.com/5459225/61748094-6475e800-ad6d-11e9-9c8b-ec916777163d.png)